ipatests/test_integration/test_automember.py

```diff
@@ -246,6 +251,11 @@ def test_modify_host_entry_unmembership_disabled(self):
             assert self.is_host_member_of_hostgroup(host2, hostgroup2)
             assert not self.is_host_member_of_hostgroup(host2, hostgroup1)
 
+            msg = ('IMPORTANT: In case of a high number of users, hosts or '
+                   'groups, the operation may require high CPU usage.')
+            result = self.master.run_command(['ipa', 'automember-rebuild'])
```

Comment:
1. no need to run the command a second time. You just need to store the command result.
2. The same message is used in 2 different places. IMO a constant could be defined in the test class and used in the 2 tests.